docs/user_guide/installation.rst

```diff
@@ -164,6 +228,24 @@ Once you have verified that your remote configuration works, try compiling a sim
    (venv) curl https://raw.githubusercontent.com/siliconcompiler/siliconcompiler/main/docs/user_guide/examples/heartbeat.v > heartbeat.v
    (venv) sc heartbeat.v -remote
 
+
+Run Locally
+^^^^^^^^^^^
+
+.. _Run local:
+
+If you wish to run locally, you will need to install some external tool dependencies to start. Take a look at `External Tools`_ to start with Surelog, Yosys, OpenRoad and KLayout as a minimum to run.
```

Comment:
I think this should be clarified to say that those tools are required for as a minimum for a ASIC flow